pkg/cloud/services/securitygroup/securitygroups.go

```diff
@@ -53,7 +53,7 @@ const (
 	IPProtocolICMP = "icmp"
 
 	// IPProtocolICMPv6 is how EC2 represents the ICMPv6 protocol in ingress rules.
-	IPProtocolICMPv6 = "58"
+	IPProtocolICMPv6 = "icmpv6"
```

Comment:
Could this change break existing clusters that are already using the previous value?